q_learning.py

```diff
@@ -12,93 +12,101 @@
 import math
 import argparse
 import os
+import tensorflow as tf
 import numpy as np
 import matplotlib.pyplot as plt
 from bayesianapproximator import *
 from BNNApproximation import BNNApproximation
 from ExperimentDescription import ExperimentDescription
 import utils.registry as registry
 from pickle import dump
+tf.enable_eager_execution()
 
-def runExperiment(env, num_episodes, agent):
```

Comment:
I can't tell, are these all whitespace changes? They better not be tabs :stuck_out_tongue: